src/query/query_opfunc.c

```diff
@@ -10753,6 +10953,116 @@ qdata_json_remove (THREAD_ENTRY * thread_p, FUNCTION_TYPE * function_p, VAL_DESC
   return error_status;
 }
 
+static int
+qdata_json_array_append (THREAD_ENTRY * thread_p, FUNCTION_TYPE * function_p, VAL_DESCR * val_desc_p, OID * obj_oid_p,
```

Comment:
maybe we should create a "template" qdata_function_name (e.g. qdata_json_array) function that also has a function pointer of type db_function_name in the arguments (e.g. db_json_array) and simply use that instead of all this repeated code. can you please do that for all functions?